README.md

```diff
@@ -21,3 +21,19 @@ You can reach the maintainers of this project at:
 ### Code of conduct
 
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+
+## Build
+
+Please make sure you have the REGISTRY environment variable set.
+For local builds it can be set to anything.
+For image builds it determines the location of your image.
+The REGISTRY defaults to `gcr.io/$(shell gcloud config get-value project)`.
+
+## Published images
+
+Alternatively, you can use the published images in your manifests. The latest
+images are pushed to the following Kubernetes production repos:
+
+- us.gcr.io/k8s-artifacts-prod/storage-migrator
```

Comment:
Is it too late to change storage-migrator to storage-version-migrator?

Reply:
I didn't use `storage-version-migrator` because it exceeds the [18-characters limit](https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/README.md#creating-staging-repos). Do you have any other suggestions? Or we can ask `k8s.io` folks if they can make an exception.

Changing the name means creating a new repo. I don't think it's too late to change, we can always create a new one following the same process. Then we can update the README here to use the new repo.

Reply:
Regarding the project name length, I think it's a hard limit of 18 characters: https://github.com/kubernetes/k8s.io/pull/399